[CO-3164] Specify NULL is not supported in CSV upload. (#8672)
braze-inc/braze-docs: _docs/_user_guide/personalization_and_dynamic_content/catalogs/catalog.md

5. Click **Process Catalog** to create the catalog.

{% alert note %}

> This data type cannot be edited after you set up your catalog. In addition, `NULL` value is not supported in CSV upload therefore will be treated as a string.

Reviewer: ```suggestion
This data type cannot be edited after you set up your catalog. In addition, a `NULL` value isn't supported in CSV upload and will be treated as a string.
```
